config/celery.py

```
@@ -8,8 +8,12 @@
 app.autodiscover_tasks()
 
 # 태스크를 명시적으로 import하여 등록 보장
+# Django 앱이 로드된 후에 import하도록 지연 (순환 import 방지)
+# companies.tasks는 companies/tasks/__init__.py에서 자동으로 import되므로
+# autodiscover_tasks()가 자동으로 찾습니다.
 try:
     import core.tasks.price_sync  # noqa: F401
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)